### DISCUSSION TITLE
Commands fabric-ca-client

### DISCUSSION THREAD

**Discussion by douglasjava** [hyperledger/fabric-samples#1061] [Q&A]
Good evening, does this fabric-ca-client CLI have documentation? with it is possible to obtain information from the network and manage it?

**Comment by denali49** [comment]
Look in the fabric-ca repo.

On Wed, Jun 21, 2023 at 7:56 PM Douglas Dias ***@***.***>
wrote:

> Good evening, does this fabric-ca-client CLI have documentation? with it
> is possible to obtain information from the network and manage it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hyperledger/fabric-samples/discussions/1061>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH5FSLVZ5LLF4WC2DU7XJEDXMOJ4FANCNFSM6AAAAAAZPPPLNU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Chris Gabriel, Founder
Hyperchain Labs
***@***.***
918-397-4078

hyperchainlabs.com
Twitter @HyperchainLabs
